### PR TITLE
fix eslint issues (formatting changes only)

### PR DESCRIPTION
**Pull request by ndelangen**
telescoped off https://github.com/storybookjs/storybook/pull/15308

### CODE REVIEW

File: addons/docs/src/frameworks/react/jsxDecorator.test.tsx [hunk 1]
```diff
@@ -108,34 +108,32 @@ describe('renderJsx', () => {
...
+      <div>{props.children}</div>
+    ));
 
     expect(renderJsx(<MyExoticComponent>I'm forwardRef!</MyExoticComponent>, {}))
       .toMatchInlineSnapshot(`
-        <MyExoticComponent>
-          I'm forwardRef!
-        </MyExoticComponent>
-      `);
+      <[object Object]>
```
Comment (shilman) [inline addons/docs/src/frameworks/react/jsxDecorator.test.tsx:117]: something is stripping off displayName??

File: .eslintrc.js [hunk 2]
```diff
@@ -68,5 +69,12 @@ module.exports = {
         'react-hooks/rules-of-hooks': 'off',
       },
     },
+    {
+      files: ['**/*.stories.*'],
+      rules: {
+        'react/display-name': 'off',
```
Comment (shilman) [inline .eslintrc.js:75]: Is this causing problems somehow?

File: addons/docs/src/frameworks/react/jsxDecorator.test.tsx (hunk 1 above)
Comment (ndelangen) [reply addons/docs/src/frameworks/react/jsxDecorator.test.tsx:117]: Yeah, but we don't know what..

File: .eslintrc.js (hunk 2 above)
Comment (ndelangen) [reply .eslintrc.js:75]: Wouldn't know how this might causes issues really.

**Comment by shilman** [comment]
closing this alongside #15308

**Pull request closed without merging**